tests/integration/smcp-httpbin-test.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+
+# Copyright 2022 Red Hat, Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Exit immediately for non zero status
+set -e
+# Check unset variables
+set -u
+# Print commands
+set -x
+
+set -o pipefail
+
+WD=$(dirname "$0")
+WD=$(cd "$WD"; pwd)
+ROOT="$(git rev-parse --show-toplevel)"
+CR="${ROOT}/deploy/examples/maistra_v2_servicemeshcontrolplane_cr_minimal.yaml"
+BRANCH="${BRANCH:-maistra-2.3}"
+HTTPBIN="https://raw.githubusercontent.com/maistra/istio/${BRANCH}/samples/httpbin/httpbin.yaml"
+SLEEP="https://raw.githubusercontent.com/maistra/istio/${BRANCH}/samples/sleep/sleep.yaml"
+DPNS="default"
+
+create-control-plane() { # creates a SMCP with a mini CR
+    local CPNS="${CPNS:-istio-system}"
+    kubectl get ns "${DPNS}" >/dev/null 2>&1 || kubectl create namespace "${DPNS}"
+    kubectl get ns "${CPNS}" >/dev/null 2>&1 || kubectl create namespace "${CPNS}"
+
+    sed "s:#- bookinfo:- ${DPNS}:g" "${CR}" | kubectl apply -n "${CPNS}" -f -
```

Comment:
Could you add a comment here that you are setting `${DPNS}` namespace as a member of SMMR?

Reply:
+1

BTW what does `DPNS` mean? Can you rename it to `APP_NS` or `MEMBER_NS` or similar, so it's immediately clear what it is? 

`CPNS` is pretty clear, but it would probably be clearer if it was called `SMCP_NS`. We also use the name `MESH_NS` in the code, but that's not as clear as `SMCP_NS`.